test/percy/spec.jsx

```diff
@@ -48,11 +51,45 @@ const PropValue = styled.span`
   box-sizing: border-box;
 `;
 
-const Box = styled.div`
-  background-color: ${(props) =>
-    props.backgroundColor ?? designTokens.colorSurface};
-  }};
-`;
+const Box = (props) => {
+  const { isNewTheme } = useTheme(
+    props.boxId ? () => document.getElementById(props.boxId) : undefined
+  );
+  return (
+    <div
+      css={css`
+        background-color: ${props.backgroundColor ?? designTokens.colorSurface};
+        margin-bottom: 20px;
+
+        ${isNewTheme &&
+        css`
+          font-family: 'Inter', system-ui;
+          font-size: 16px; // This won't ever work since we use 'rem' units in components, so the font size will be calculated based on the font-size set in the <html> element
+        `}
```

Comment:
Hey @commercetools/shield-team-fe 

I have a question to you. Apparently this setup won’t ever work (at least I can’t think of how I can make it work). We are changing `font-size` in the new theme in the root element and all components use `rem` units. Using this approach with nested <ThemeProvider> contexts in the VRTs makes no sense. Even if we change `font-size`  in the local <ThemeProvider> context it won’t ever be taken into account in the components. 

<img width="352" alt="Screenshot 2023-03-23 at 10 01 52" src="https://user-images.githubusercontent.com/49066275/227155974-7bbf3b7c-3181-4bfe-9257-3752d38a294c.png">

Would you be ok with using only the new theme in the VRTs?

Reply:
Thanks a lot for looking into this, @kark 🙇 

My suggestion here would be to move forward with this approach and let me explain why: I think VRTs are actually used as a baseline to check we don't break something. Breaking something means a new screenshot **looks different** than the previous stored one, but it actually does not matter too much **how** it looks like. I mean, yes we want the screenshots to look how we expect but for the check sake, it just matters it a new one looks different than the previous one.

Given this is a temporary solution until we finally go back to having just one theme, I think we could tolerate the new one to not use the expected font sizes.

If we ever support more themes, now we now this approach would not be suitable and we might need to have more _credits_ in Percy to run different sets of screenshots.